package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Isomorphic React boilerplate",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "start": "APP_ENV=development node index",
     "eb-start": "npm run dist && APP_ENV=development node index",
     "dist": "NODE_ENV=production webpack --config webpack.config.js",
     "postinstall": "npm run dist",
-    "test": "node_modules/.bin/mocha test/helpers/browser.js test/unit/*.test.js",
+    "test": "node_modules/.bin/mocha test/helpers/browser.js test/unit/$TEST_FILE",
```

Comment:
Update the script to test a single unit test file. The way to test it is utilizing the env variable `TEST_FILE`. I feel it is more simple than typing the whole file path. But it might be a bit funky way to use env variable. Related instruction can be found in `README`. Open to discuss a better way as well :-)